.github/workflows/ci-solana-contract.yml

```diff
@@ -0,0 +1,33 @@
+name: Test Solana Contract
+
+on:
+  pull_request:
+    paths:
+      - target_chains/cosmwasm/**
+      - pythnet/pythnet_sdk/**
+  push:
+    branches:
+      - main
+    paths:
+      - target_chains/cosmwasm/**
+      - pythnet/pythnet_sdk/**
```

Comment:
should both of these sets of paths also include target_chains/solana?

Reply:
I can't believe I forgot to replace this